recipes/cclib/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "cclib" %}
+{% set version = "1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 636dfbc192c0a95079aaba3fb0c4872222c2a8d500f28d681bc0522fe81dc49a
+
+build:
```

Comment:
Can this be `noarch: python`?

Reply:
I don't see why it couldn't. It agrees with every point mentioned [here](https://conda-forge.org/docs/meta.html#building-noarch-packages). I have tested it locally and seems to work correctly. I have added it in the last commit.